WordPress/src/androidTest/java/org/wordpress/android/e2etests/ComposeTests.java

```diff
@@ -24,17 +24,16 @@
     @Rule
     public ActivityTestRule<WPLaunchActivity> mActivityTestRule = new ActivityTestRule<>(WPLaunchActivity.class);
 
-    private TestData mData = new TestData();
-    private String mUserEmail = mData.getEmail();
-    private String mUserPassword = mData.getPassword();
+    String mUsername = BuildConfig.ESPRESSO_USERNAME;
+    String mPassword = BuildConfig.ESPRESSO_PASSWORD;
```

Comment:
I wonder if `mUsername`, `mPassword` are mutable by design. Can we define them as constants instead? Global variables in tests are a possible reason for hidden bugs and can make the tests flaky if a test mutates such a variable by accident.